MANIFEST.in

```diff
@@ -1,12 +1,16 @@
-prune clients
-prune tests
```

Comment:
Doesn't look that way, my sdist archive doesn't contain tests. Is your version of setuptools including it?